src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -409,9 +410,22 @@
                         </a>
                       {% endif %}
 
-                      <!-- Event address -->
-                      {% if eventReference.entity.fieldLocationHumanreadable != empty %}
-                        <p class="vads-u-margin--0">{{ eventReference.entity.fieldLocationHumanreadable }}</p>
+
+                      <!-- Online event link -->
+                      {% if eventReference.entity.fieldUrlOfAnOnlineEvent.uri %}
+                        <a
+                          href="{{ eventReference.entity.fieldUrlOfAnOnlineEvent.uri }}"
+                          onclick="recordEvent({ 'event': 'clp-link-click', 'clp-section-category': 'Events', 'clp-section-title': '{{ fieldClpEventsHeader | escape }}', 'clp-total-sections': '{{ clpTotalSections }}', 'clp-click-label': '{{ eventReference.entity.fieldUrlOfAnOnlineEvent.uri | escape }}' });"
+                          rel="noreferrer noopener"
+                          target="_blank"
+                        >
```

Comment:
I think we can omit the `rel` and `target`. We have a build step that will add that automatically if the link's domain is considered external

Reply:
Would an online event link ever be on va.gov? Just wondering if it's necessary to remove

Reply:
not certain but we have events sections on va.gov so maybe? i figured most event links would be to www.va.gov

Reply:
Removed 👍